projects/managed-hubs/support.md

```diff
@@ -1,118 +1,214 @@
+Trusted community members that perform common administrative operations on a hub that do not require intervention from a [Hub Engineer](roles:hub-engineer).
+[Community Representatives](roles:community-representative) are the first Hub Administrators, and they may add new Hub Administrators via the JupyterHub interface.
+They are able to add users, start/stop servers, and generally have more control over operations on the hub.
 
-- Be placed in our [Sprint Board](coordination:sprint-board).
-- Include a reference to any FreshDesk tickets if they exist.
-- Be prioritized over other backlog items for that sprint.
+Their responsibilities include:
+
+- Provide support to users of a hub for common problems that don't require a Hub Engineer to resolve.
+- Add new users to a hub, including administrative users.
+- Surface major issues or requests to the Community Representative(s).
 
 ## Communication channels
 
-We use two Slack channels to discuss support-related questions on the 2i2c hub:
+### External communication
 
-- The [#support-freshdesk](https://2i2c.slack.com/archives/C028WU9PFBN) is for discussing support tickets on Freshdesk.
-  Open a Slack thread for each support ticket, to have conversation around it amongst the team.
-  This channel is private because there’s an assumption of privacy when we discuss community-specific support matters.
-- The [#hub-support](https://2i2c.slack.com/archives/C01DB2JRP8W) channel is for discussing more general support-related issues relating to operating the 2i2c Hubs, and not not related to a specific FreshDesk ticket.
-  This channel is public, so can be useful if we wish to invite conversation from non 2i2c team members.
+We use these channels for communicating with external stakeholders like Community Representatives:
 
-## Lifecycle of a support request
+- **[support@2i2c.org](mailto:support@2i2c.org)** is our point-of-contact for all support-related external communication.
+- **[The 2i2c FreshDesk account](https://2i2c.freshdesk.com/)** is where we track all support requests and _respond_ to emails sent to `support@2i2c.org`.
+- **{doc}`the "Get Support" page <docs:support>`** provides guidance that communities may follow to get support.
 
+### Internal communication
 
-Below is a summary of the major steps that make up the support process, from beginning to end of a ticket.
+We have a few channels for communicating around support requests:
 
-:::{note}
-We should try to respond to all support-related communications within one working day.
-:::
+- Our [FreshDesk account](https://2i2c.freshdesk.com/a/) allows for internal team communication via the {guilabel}`Add Note` button. This can be useful for sharing quick internal updates.
+- [Issues with the {guilabel}`support` label](https://github.com/2i2c-org/infrastructure/issues?q=is%3Aopen+label%3Asupport+sort%3Aupdated-desc) are where we track support requests related to {term}`Change Request`s and {term}`Guidance Request`s.
```

Comment:
Maybe adding a note about starting the request via the support email instead of creating an issue on GH.